dkg/runner.go

```diff
@@ -51,22 +54,28 @@ func (r *Runner) ProcessMsg(msg *SignedMessage) (bool, map[types.OperatorID]*Sig
 			if err != nil {
 				return false, nil, errors.Wrap(err, "could not generate deposit data")
 			}
+
 			r.DepositDataRoot = root
 
 			// sign
+			sig := r.KeyGenOutput.Share.SignByte(root)
```

Comment:
why was this replaced?

Reply:
The deposit data needs to be signed with the bls key, which in here means we need to use the share to sign. I don't understand what's the purpose of the original function `SignETHDepositRoot(root, r.Operator.ETHAddress)` but share is not related to `ETHAddress`.

Note: At the moment `SignETHDepositRoot` is not used at all.